contained-innersource-enables-collaborative-product-development.md

```diff
@@ -0,0 +1,35 @@
+## Title  
+Contained inner source enables collaborative product development
+
+## Context
```

Comment:
Should we change it so that it matches something that you've seen work? Otherwise, this would just be a pattern idea. Did I misunderstand our discussion in the dim sum dinner? Or mis-document it?